bytes/bytes/database/sql_meta_repository.py

```diff
@@ -195,6 +200,11 @@ def __init__(self, cls: Type[SQL_BASE], **kwargs):  # type: ignore
         super().__init__(f"The object of type {cls} was not found for query parameters {kwargs}")
 
 
+class MetaIntegrityError(Exception):
+    def __init__(self, message: str):
+        super().__init__(message)
```

Comment:
Technically, this initializer is unnecessary since it will forward all arguments by default to the parent's `__init__`. Unless you really want to force the user to specify an error message. Imho I would leave it out since the default (str) representation of an exception is the passed arguments, which is often always a message and used by others as such